src/extension.ts

```diff
@@ -103,7 +103,7 @@ async function startDebug(_: ExtensionContext, config: DebugConfig): Promise<boo
     let rawStdErr: string | undefined = undefined;
     try {
         const args = ['debug', '-I', '-b', config.board.fqbn, config.sketchPath, '--format', 'json'];
-        const { stdout, stderr } = spawnSync(config.cliPath, args, { encoding: 'utf8' });
+        const { stdout, stderr } = spawnSync(config.cliDaemonAddr, args, { encoding: 'utf8' });
```

Comment:
This is unrelated to the language server I think this should remain `cliPath` (also I guess `spawnSync` wants the path of an executable)